django_rq/management/commands/rqworker.py

```diff
@@ -66,12 +68,26 @@ def add_arguments(self, parser):
                             help='Default worker timeout to be used')
         parser.add_argument('--sentry-dsn', action='store', default=None, dest='sentry-dsn',
                             help='Report exceptions to this Sentry DSN')
+        parser.add_argument('--autoreload', action='store_true', dest='use_reloader',
+                            help='Auto-reloads the worker code.')
 
         if LooseVersion(get_version()) >= LooseVersion('1.10'):
             parser.add_argument('args', nargs='*', type=str,
                                 help='The queues to work on, separated by space')
 
     def handle(self, *args, **options):
+        if options['use_reloader']:
+            if not settings.DEBUG:
```

Comment:
i would put this block above, on line 79. you should be warning always if you’re not on debug... no?

Reply:
Sorry, I don't understand. Autoreload should be used in development only. Did you want to merge these two `if`s together? I can't do that because there are more things going on than just showing the warning. Please, explain

Reply:
Oh i see.. i completely read that wrong -- i apologize. You're correct. Nothing needs to happen here.